Sources/Logging/Loggable.swift

```diff
@@ -2,6 +2,13 @@ import Foundation
 import MetricKit
 
 protocol Loggable {
+    
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Logging/Loggable.swift;
  Line: 5;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)